tests/previewGroup.test.tsx

```diff
@@ -96,6 +87,28 @@ describe('Preview', () => {
     expect(
       wrapper.find('.rc-image-preview .rc-image-preview-switch-left-disabled').get(0),
     ).toBeTruthy();
+
+    act(() => {
+      const event = new KeyboardEvent('keydown', { keyCode: 39 });
```

Comment:
also use `rc-util` KeyCode in test case.

Reply:
Updated! Thanks for remind.